src/components/tasks/card/index.tsx

```diff
@@ -603,7 +603,9 @@ const Card: FC<CardProps> = ({
                 </span>
                 {isEditable
                     ? isUserAuthorized && (
-                          <div className={classNames.assignedToSection}>
+                          <div
+                              className={`${classNames.assigneeSuggestionInput} ${classNames.assignedToSection}`}
```

Comment:
These class names should be using BEM convention used mostly in the app I think.

Noticed few components having different naming convention, which should be changed to BEM(out of scope of this PR).